AppCenterIdentity/AppCenterIdentity/Internals/MSIdentityPrivate.h

```diff
@@ -19,9 +19,9 @@ static NSString *const kMSIdentityETagKey = @"MSIdentityETagKey";
 @property(nonatomic, nullable) MSALPublicClientApplication *clientApplication;
 
 /**
- * The user access token to be used for user identifier.
+ * The ID token to be used for user identifier.
  */
-@property(nonatomic, nullable) NSString *accessToken;
+@property(atomic, nullable, copy) NSString *idToken;
```

Comment:
No need to specify `atomic` explicitly, it's atomic by default.

Reply:
True, I can change. We are explicit about saying `atomic` in a lot of places, changes that i have made recently in other PRs, because i don't want anyone to change them to nonatomic, so it was a defensive thing i guess.

Reply:
I see. Good thinking! Let's leave it there. Maybe even add a comment? (leaving that up to you)

Reply:
I'll let the `atomic` being explicit speak for itself :)